doc/tutorials/introduction/table_of_content_introduction.markdown

```diff
@@ -163,3 +163,11 @@ Additionally you can find very basic sample source code to introduce you to the
     _Author:_ Maksim Shabunin
 
     This document describes some aspects of 2.4 -> 3.0 transition process.
+
+-   @subpage tutorial_building_tegra_cuda
```

Comment:
Maybe it will be better to move new item higher: before or after ARM cross-compilation tutorial.